angular/src/app/workflows/workflow-list-single/workflow-list-single.component.html

```diff
@@ -58,5 +58,10 @@ <h3 class="flex vertical-center">
         </mat-action-row>
       </mat-expansion-panel>
     </mat-accordion>
+    <ddap-paginator *ngIf="pageToken.length"
+                    [nextPage]="workflowRunsResponse.next_page_token"
+                    [paginationType]="paginationType"
```

Comment:
`[paginationType]="PaginationTypes.UNIDIRECTION"` it would be more readable, no need to create a variable for it